src/indicators.ts

```diff
@@ -100,6 +108,7 @@ export class Indicators {
 
     private filterCoverage(section: Section, coverageLines: ICoverageLines): ICoverageLines {
         section.lines.details.forEach((detail) => {
+            if (detail.line < 0) { return ; }
```

Comment:
FYI these changes are in my other PR https://github.com/ryanluker/vscode-coverage-gutters/pull/107/files#diff-fc82abb2ab9c0fe32935ce85f70da9a7R103 but no harm getting them in early 😄